db/src/db.rs

```diff
@@ -331,6 +331,24 @@ pub fn ensure_current_version(conn: &mut rusqlite::Connection) -> Result<DB> {
     }
 }
 
+pub trait SQLValueType {
+    fn value_type_tag(&self) -> i32;
+}
+
+impl SQLValueType for ValueType {
+    fn value_type_tag(&self) -> i32 {
+        match *self {
+            ValueType::Ref =>      0,
+            ValueType::Boolean =>  1,
+            ValueType::Instant =>  4,
+            ValueType::Long =>     5,
```

Comment:
nit: we've been fairly consistent about including the note about overlapping tags more or less every place it could be included.  Follow suit?